be/src/controlers/health.router.ts

```diff
@@ -0,0 +1,17 @@
+import {Request, Response, Router} from 'express';
+import PersistenceService from "../services/persistence-service";
+
+const router: Router = Router();
+
+router.get('/health', async (req: Request, res: Response) => {
+    console.log("HEALTH")
+    const health_check = await PersistenceService.pingDB()
+
+    if (health_check) {
+        res.json({statusCode: 200, greeting: "Hello World"});
```

Comment:
Why returning `Hello World` in a ping response?
```suggestion
        res.status(200);
```

Reply:
@ja-fra is returing 200 sufficient? Is anything of
https://docs.mongodb.com/v4.2/reference/command/serverStatus/
be needed?

Reply:
For a start, response code `200` / ´5xx` is sufficient.

In future we could add more details, especially if we start to have more component integrated into the healthcheck.